direct/common/subsample.py

```diff
@@ -243,6 +246,83 @@ def mask_func(
         return torch.from_numpy(self._reshape_and_broadcast_mask(shape, mask))
 
 
+class CartesianRandomMaskFunc(FastMRIRandomMaskFunc):
+    r"""Cartesian random vertical line mask function.
+
+    Similar to :class:`FastMRIRandomMaskFunc, but instead of center fraction (`center_fractions`) representing
+    the fraction of center lines to the original size, here, it represents the actual number of center lines.
```

Comment:
It's not deriving from the FastMRI ones, isn't there a better way to do this? As in both are subclassing something more general?

Reply:
Both Cartesian and FastMRI classes inherit from a single class now